x-pack/plugins/security_solution/public/resolver/mocks/resolver_tree.ts

```diff
@@ -6,8 +6,49 @@
 
 import { mockEndpointEvent } from './endpoint_event';
 import { ResolverTree, SafeResolverEvent } from '../../../common/endpoint/types';
+import { relatedEventsPaginationSize } from '../data_access_layer/factory';
 import * as eventModel from '../../../common/endpoint/models/event';
 
+export function mockTreeWithOneNodeAndTwoPagesOfRelatedEvents({
```

Comment:
I don't think we should couple this to the page size. The API ultimately defines the page size. The mock should be able to arbitrarily decide the number of items coming back each time.